dev/spark-test-image/python-minimum/Dockerfile

```diff
@@ -61,7 +61,7 @@ RUN apt-get update && apt-get install -y \
     wget \
     zlib1g-dev
 
-ARG BASIC_PIP_PKGS="numpy==1.22.4 pyarrow==15.0.0 pandas==2.2.0 six==1.16.0 scipy scikit-learn coverage unittest-xml-reporting"
+ARG BASIC_PIP_PKGS="numpy==1.22.4 pyarrow==15.0.0 pandas==2.3.3 six==1.16.0 scipy scikit-learn coverage unittest-xml-reporting"
```

Comment:
Please revert this because the minimum Pandas version is still 2.2.0.

https://github.com/apache/spark/blob/e837ff9707ef5d08cde53c338dfaf7d81a5c55aa/python/packaging/classic/setup.py#L153